### ISSUE TITLE
Phreaknet install issue as of 06-03-2023 [Makefile:396: res] Error 2

### ISSUE THREAD

**Issue by chrisvictor** [InterLinked1/phreakscript#17]
Errors below:
   [CCi] res_convert.i -> res_convert.o
   [CCi] res_crypto.i -> res_crypto.o
   [CCi] res_curl.i -> res_curl.o
res_crypto.c: In function â€˜try_load_key.israâ€™:
res_crypto.c:199:35: error: â€˜%sâ€™ directive output may be truncated writing between 5 and 2147483645 bytes into a region of size 255 [-Werror=format-truncation=]
  199 |  snprintf(ffname, sizeof(ffname), "%s/%s", dir, fname);
      |                                   ^~~~~~~
res_crypto.c:199:2: note: â€˜snprintfâ€™ output 7 or more bytes (assuming 2147483647) into a destination of size 256
  199 |  snprintf(ffname, sizeof(ffname), "%s/%s", dir, fname);
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   [CCi] res_deadlock.i -> res_deadlock.o
cc1: all warnings being treated as errors
make[1]: *** [/usr/src/asterisk-20.3.0/Makefile.rules:150: res_crypto.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:396: res] Error 2
root@TEST-ASL30-K6CV:/# 
root@TEST-ASL30-K6CV:/# phreaknet info
Hostname: TEST-ASL30-K6CV
Raspbian GNU/Linux 11 (bullseye)
Linux TEST-ASL30-K6CV 6.1.21-v7+ #1642 SMP Mon Apr  3 17:20:52 BST 2023 armv7l GNU/Linux
Package Manager: apt-get
DAHDI Tools Version - 3.2.0
DAHDI Version: 3.2.0
PhreakScript 0.3.3 (2023-05-25)

**Comment by InterLinked1** [comment]
Is this on 32-bit? Always specify your bitness and kernel/OS.
This isn't PhreakNet/PhreakScript related, this is an upstream Asterisk issue due to changes somebody made there a few months back.
I think you reported this before, actually, and I passed it along upstream. Of course, it now needs to be rereported since things move to GitHub: https://issues.asterisk.org/jira/browse/ASTERISK-30450

I've opened an issue: https://github.com/asterisk/asterisk/issues/143

I'll leave this ticket open for now, and update it if there is any update upstream.
